overlays/default.nix

```diff
@@ -83,7 +83,7 @@ in
 
   alacritty_git = callOverride ../pkgs/alacritty-git { };
 
-  ananicy-cpp-rules = final.callPackage ../pkgs/ananicy-cpp-rules { };
+  ananicy-cpp-rules = callOverride ../pkgs/ananicy-cpp-rules { };
```

Comment:
Suffix it with `_git` here, and to be more explicit, it should be named `ananicy-rules-cachyos_git`. (`nyxKey` must be updated to match, no need to change the directory's name)

Reply:
Sure, I just wanted to make backwards-compatibility.

Reply:
Done, do you want me to maintain an alias?

Reply:
Nope